hax/hax/util.py

```diff
@@ -137,6 +138,12 @@ def wrapper(*args, **kwds):
                                  ('cas', Optional[Any])])
 
 
+def wait_for_event(event: Event, interval_sec) -> None:
```

Comment:
I would at least added a docstring describing that this method is actually used to perform "sleep" operation. It is not that clear from its name or signature.

Reply:
Added.